.cspell.json

```diff
@@ -405,6 +407,7 @@
     "datafiles",
     "dataholder",
     "datastart",
+    "datatypes",
```

Comment:
I think this should be replaced by `data types`.

Reply:
Yes, right! https://github.com/tier4/autoware-spell-check-dict/pull/280/commits/2590739fda75ef8df77d3fe911ea5e54419ebaa7